laravel/resources/views/instances.blade.php

```diff
@@ -1,149 +1,149 @@
+                        </td>
+                        <td class="col-lg-3">
+                            {{ $instance->host }}
+                            @if($instance->is_ssh)
+                                <span class="badge text-bg-success ms-2" data-bs-toggle="tooltip" data-bs-html="true"
+                                      title="{{ $instance->serverquery_port }} (TCP)"
+                                      id="instance-port-ssh-badge">SSH
+                                </span>
+                            @else
+                                <span class="badge text-bg-warning ms-2" data-bs-toggle="tooltip" data-bs-html="true"
+                                      title="{{ $instance->serverquery_port }} (TCP)"
+                                      id="instance-port-raw-badge">RAW
+                                </span>
+                            @endif
+                        </td>
+                        <td class="col-lg-1">
+                            {{ $instance->voice_port }}
+                        </td>
+                        <td class="col-lg-1">
+                            {{ $instance->client_nickname }}
+                        </td>
+                        <td class="col-lg-2">
+                            @if (is_null($instance->process))
+                            @can('start instances')
+                                <a href="{{ route('instance.start', ['instance_id' => $instance->id]) }}"><i class="fa-solid fa-play text-success me-2 fa-lg"></i></a>
+                            @endcan
+                            @else
+                            @can('stop instances')
+                                <a href="{{ route('instance.stop', ['instance_id' => $instance->id]) }}"><i class="fa-solid fa-power-off text-warning me-2 fa-lg"></i></a>
+                            @endcan
+                            @endif
+                            @can('restart instances')
+                                <a href="{{ route('instance.restart', ['instance_id' => $instance->id]) }}"><i class="fa-solid fa-rotate-left text-warning me-2 fa-lg"></i></a>
+                            @endcan
+                            @can('edit instances')
+                                <a href="#modalEditInstance-{{$instance->id}}" data-bs-toggle="modal" data-bs-target="#modalEditInstance-{{$instance->id}}"><i class="fa-solid fa-pencil text-primary me-2 fa-lg"></i></a>
+                            @endcan
+                            @can('delete instances')
+                                <a href="{{ route('instance.delete', ['instance_id' => $instance->id]) }}"><i class="fa-solid fa-trash text-danger me-2 fa-lg"></i></a>
+                            @endcan
+                        </td>
+                    </tr>
+                    @endforeach
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        @endif
     </div>
-</div>
+    <script type="module">
```

Comment:
[CLARIFICATION]

Shouldn't we import here the respective outsourced code instead of writing it here again?